### PR TITLE
Implemented StarTime and Uptime

### PR DESCRIPTION
**Pull request by Silv3rcircl3**
The JVM version has this two properties which I need in the ActorSystemSpec.

### CODE REVIEW

File: src/core/Akka/Actor/ActorSystem.cs [hunk 1]
```diff
@@ -74,6 +74,24 @@ public abstract class ActorSystem : IActorRefFactory, IDisposable
...
+
+        /// <summary>
+        /// Up-time of this actor system in seconds.
+        /// </summary>
+        public long Uptime => (MillisecondsSinceEpoch() - StartTime) / 1000;
+
+        /// <summary>
+        /// Returns the time in milliseconds since the epoch
+        /// </summary>
+        private static long MillisecondsSinceEpoch()
```
Comment (Silv3rcircl3) [inline src/core/Akka/Actor/ActorSystem.cs]: Should this be maybe moved into Utils?

File: src/core/Akka/Actor/ActorSystem.cs [hunk 2]
```diff
@@ -74,6 +74,24 @@ public abstract class ActorSystem : IActorRefFactory, IDisposable
...
+        /// Start-up time in milliseconds since the epoch.
+        /// </summary>
+        public long StartTime => _startTime;
+
+        private readonly long _startTime = MillisecondsSinceEpoch();
+
+        /// <summary>
+        /// Up-time of this actor system in seconds.
+        /// </summary>
+        public long Uptime => (MillisecondsSinceEpoch() - StartTime) / 1000;
```
Comment (rogeralsing) [inline src/core/Akka/Actor/ActorSystem.cs]: Is seconds really the format one would expect from this?
Why not Timespan?

Comment (Silv3rcircl3) [inline src/core/Akka/Actor/ActorSystem.cs]: I have used the same signature as the scala version but would also prefer Timespan.

File: src/core/Akka/Actor/ActorSystem.cs (hunk 1 above)
Comment (Aaronontheweb) [inline src/core/Akka/Actor/ActorSystem.cs]: You should use the methods we have in the `MonotonicClock` under utils and see if that suits your needs here. If it doesn't, then yes you should move this into a utility class in that namespace.

File: src/core/Akka/Actor/ActorSystem.cs (hunk 2 above)
Comment (Aaronontheweb) [inline src/core/Akka/Actor/ActorSystem.cs]: Let's use a `TimeSpan` here - it's a value type so there shouldn't be any allocations.

**Comment by Silv3rcircl3** [comment]
@Aaronontheweb @rogeralsing updated

**Comment by rogeralsing** [comment]
LGTM, I'm restarting the multi node tests before merge

**Comment by Aaronontheweb** [comment]
I'm going to fix that racy one soon, now that I have better logs

**Comment by Aaronontheweb** [comment]
@Silv3rcircl3 this will need to be rebased on dev due to #1639 being merged in.

Bear in mind you'll need to run the unit tests locally first to generate a new .approved.txt file per the new process we're using to control how the public API gets modified. I outlined this here: http://getakka.net/docs/akka-developers/public-api-changes

Could you give that a try?

File: src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt [hunk 3]
```diff
@@ -3765,7 +3767,7 @@ namespace Akka.Routing
     }
     public class DefaultResizer : Akka.Routing.Resizer, System.IEquatable<Akka.Routing.DefaultResizer>
     {
-        public DefaultResizer(int lower, int upper, int pressureThreshold = 1, double rampupRate = 0.2, double backoffThreshold = 0.3, double backoffRate = 0.1, int messagesPerResize = 10) { }
+        public DefaultResizer(int lower, int upper, int pressureThreshold = 1, double rampupRate = 0,2, double backoffThreshold = 0,3, double backoffRate = 0,1, int messagesPerResize = 10) { }
```
Comment (Silv3rcircl3) [inline src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt]: I don't know were this is coming from since I didn't have changed this class.

**Comment by Silv3rcircl3** [comment]
@Aaronontheweb updated

**Comment by Silv3rcircl3** [comment]
It's working on my machine :wink: 
image

**Comment by Silv3rcircl3** [comment]
Fixed Api approve test.

**Pull request merged**